src/main/java/org/matsim/pt2matsim/gtfs/GtfsConverter.java

```diff
@@ -396,14 +396,23 @@ private void loadCalendarDates() {
 			String[] line = reader.readNext();
 			while(line != null) {
 				Service currentService = services.get(line[col.get(GTFSDefinitions.SERVICE_ID)]);
-				if(currentService != null) {
-					if(line[col.get(GTFSDefinitions.EXCEPTION_TYPE)].equals("2"))
-						currentService.addException(line[col.get(GTFSDefinitions.DATE)]);
-					else
-						currentService.addAddition(line[col.get(GTFSDefinitions.DATE)]);
-				} else {
-					throw new RuntimeException("Service id \"" + line[col.get(GTFSDefinitions.SERVICE_ID)] + "\" not defined in calendar.txt");
+
+				if(currentService == null) {
+					currentService = new Service(
+							line[col.get(GTFSDefinitions.SERVICE_ID)],
+							new boolean[] { false, false, false, false, false, false, false },
+							"19700101", "29991231"
+					);
+
+					services.put(currentService.getId(), currentService);
+					log.warn("Service id \"" + currentService.getId() + "\" not defined in calendar.txt (only defined in calendar_dates.txt?)");
```

Comment:
This log mesage is a bit annoying (at least in this dataset), because it is shown quite often. Maybe it would make sense to make the handling of these exceptions optional through the configuration and let the user "trust" that the library handles those cases as desired.